onnx/backend/test/data/node/test_layer_normalization_2d_axis0/test_data_set_0/input_0.pb

```diff
@@ -0,0 +1 @@
+BXJ0x��?h��>��z?�j@$�?�.z��8s?b��hdӽ�9�>(�>�%�?
```

Comment:
git attributes should be set to declare all .pb files as binary, otherwise it can lead to data corruption when git interprets them as text files and messes up line ending characters.

Reply:
Is [it](https://github.com/onnx/onnx/blob/2ab133404afce34552aaccd86e7023e1fb9a60d2/.gitattributes#L1) what you're looking for?

Reply:
I changed `.gitattributes` at this [commit](https://github.com/onnx/onnx/pull/4076/commits/0fc927cc315b1386a2e15054998de5bfa1b606c1). Then, I [removed ](https://github.com/onnx/onnx/pull/4076/commits/73eeee8e90b05078eb2c91183dfdf1615e639aa4)and [added ](https://github.com/onnx/onnx/pull/4076/commits/ebfbde75451c708f13dea14270ca1f5597857a2a)back my `.pb` files. Does it look ok now?

Reply:
I have tried many combinations of `binary`, `-text`, and so on in `.gitattributes` file. Github just keeps showing the diff for `*.pb` files. Since this is a more difficult problem, I'd suggest to address it in another PR.